StochasticBarrierFunctions/src/Plots/Plots.jl

```diff
@@ -1,5 +1,8 @@
-# Plotting functions
+module Plots 
```

Comment:
We could remove the plotting part completely since its not a critical part of the toolbox. I leave the choice up to you.

Reply:
I think it would be nice to keep it - and possibly use the style from the paper. It doesn't have to be in the paper, but as part of the package, it would be nice.